publisher.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+import (
+	"fmt"
+	"time"
+	"net/http"
+	"io/ioutil"
+	"log"
+	"github.com/jlaffaye/ftp"
+	"bytes"
+	"sync"
+	"io"
+)
+
+var lock = &sync.Mutex{}
+var wg = &sync.WaitGroup{}
+
+
+func processFiles(files <- chan string) {
+	defer wg.Done()
+	conn, err := connectToFtp()
+	if err != nil {
+		log.Fatal("Unable to connect to FTP server exiting - aborting")
+	}
+	defer conn.Logout()
+	defer conn.Quit()
+	for file := range files {
+		buf, err := getFileFromFTP(file, conn)
+		if err == nil {
+			status, err := postFileToRas(file, &buf)
+			if status == 200 && err == nil {
```

Comment:
Instead of using hardcoded `200`, you could use the constants e.g. `http.StatusOK`